src/Directory.Build.props

```diff
@@ -20,7 +20,6 @@
 
   <ItemGroup>
     <PackageReference Include="MinVer" Version="2.3.1" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>

```

Comment:
Thanks, @thomaslevesque. Looks good to me. Our [instructions for using SourceLink](../../blob/master/docs/source-stepping.md) remain valid?

Reply:
> Maybe we could remove the mention that we use SourceLink; we do, but we no longer do anything explicit to use it.

Agreed!

Reply:
Removed